siliconcompiler/package/scp.py

```
@@ -0,0 +1,108 @@
+                               "in your PATH.")
+
+        # Construct the SCP command
+        command = [
+            scp,
+            "-C",  # Enable compression for faster transfers
+            "-r",  # Recursively copy directories
+            f"{self.host}:{self.host_path}",
+            self.cache_path
+        ]
+
+        result = subprocess.run(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, text=True)
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)